topics/ecology/tutorials/phylodiversity_workflow/tutorial.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: tutorial_hands_on
+
+title: Phylodiversity workflow quick tutorial
+zenodo_link: https://doi.org/10.5281/zenodo.15487657
+questions:
+- How to use the phylodiversity workflow?
+- How to construct phyloregions from occurrences species data, phylogenic data and geograpics data?
+objectives:
+- Learning how to use the phylodiversity workflow.
+- Compute endemism index
+- Create a phyloregion map
+time_estimation: ''
+tags:
+- ecology
+- taxonomic data
+contributors:
+- Morphy427
+
+---
+
+
+# Introduction
```

Comment:
```suggestion
```

(removing this makes the link previews a bit nicer when you share a link to the tutorial on e.g. social media or via chat)